assets/questions/reverseWords.js

```diff
@@ -88,6 +96,23 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
           "last reversed word."
         ].join('')
       ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.reverseString',
+      messages: [
+        "Read the instructions closely. Did you overlook something important?",
```

Comment:
The tone here's a bit off -- try to mirror other questions' first buggy output message. This tells the learner that they have a bug, but comes across slightly condescending.